taskcluster/scripts/toolchain/conda_pack_ios.sh

```diff
@@ -41,3 +53,6 @@ find ../../public/build/ -mindepth 1 -delete
 
 conda-pack -p envs/vpn -o conda-ios.tar.gz
 mv conda-ios.tar.gz  ../../public/build
+
+# remove our Pip conf, so the restrictions are back. 
+rm -f ~/.config/pip/pip.conf
```

Comment:
Note we need to manually clean this up due to: 
https://github.com/taskcluster/taskcluster/issues/6970

Otherwise this would slowly be set for all machines